dataflow/src/main/java/org/checkerframework/dataflow/analysis/Analysis.java

```diff
@@ -132,23 +140,46 @@ public void setCurrentTree(Tree currentTree) {
     /**
      * Construct an object that can perform a org.checkerframework.dataflow analysis over a control
      * flow graph. The transfer function is set later using {@code setTransferFunction}.
+     *
+     * @deprecated will be removed in next version
+     * @param env associated processing environment
      */
+    @Deprecated
     public Analysis(ProcessingEnvironment env) {
```

Comment:
There should exist a no argument constructor to replace this one, but there isn't one.